aiden-recommender/aiden_recommender/scrapers/france_travail/parser.py

```diff
@@ -0,0 +1,105 @@
+from aiden_recommender.scrapers.abstract_parser import AbstractParser
```

Comment:
you forgot to commit the __init__.py for france travail

Reply:
I think it's because you're looking at a pr into another unmegerd branch